ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -80,9 +77,8 @@ export const defaultCromwellCreateRequest: CreateAppRequest = {
     size: 50,
     diskType: DiskType.STANDARD,
   },
-  autodeleteEnabled: false,
-  // Okay to have value here because when autodeleteEnabled is false, this value is not used.
-  autodeleteThreshold: DEFAULT_AUTODELETE_THRESHOLD_MINUTES,
+  autodeleteEnabled: true,
+  autodeleteThreshold: 7 * 24 * 60, // in minutes, so this is 7 days
```

Comment:
nit: we can have a const oneDayInMinutes = 24 * 60
and then this could be 7 * oneDayInMinutes. Do you think that would be easy to understand?

Reply:
We would still need the comment here, because it's surprising to store time values in minutes.

So it would be something like 
```
autodeleteThreshold: 7 * oneDayInMinutes, // in minutes, so this is 7 days
```

I'm undecided on this one.